docs/manual/java/guide/services/index.toc

```diff
@@ -3,5 +3,6 @@ ServiceImplementation:Implementing services
 ServiceClients:Consuming services
 Test:Testing Services
 MessageSerializers:Message serializers
+ServiceInfo:Service Metadata
```

Comment:
I think it would be good to put it earlier, since `ServiceClients` kind of assumes that you know what `ServiceInfo` is already.